java/src/main/java/org/whispersystems/libsignal/IdentityKey.java

```diff
@@ -1,55 +0,0 @@
-/**
```

Comment:
IdentityKey was just a veneer around ECPublicKey, with a leading byte indicating the type of key. Despite having a provision for different key types, Signal seems to use only one key type. In our case, since the key is also the address, changing a key would change existing addresses, so it's even more unlikely that we'd ever use a new key type than Signal doing so.

So, I got rid of the key type indicator, which means we can use EcPublicKey directly, simplifying the code and avoiding some byte array copy operations.